tests/v2_0/Api/FamilyVariant/CreateFamilyVariantApiIntegration.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Akeneo\Pim\tests\v2_0\Api\FamilyVariant;
+
+use Akeneo\Pim\tests\Common\Api\ApiTestCase;
+
+class CreateFamilyVariantApiIntegration extends ApiTestCase
+{
+    public function testCreateAFamilyVariant()
```

Comment:
we should add test when the creation returns errors (422, validation error)

Reply:
done